components/WBFabric/utils/customSettings.js

```diff
@@ -82,12 +139,13 @@ function _workaround(canvas) {
 export default function (canvas) {
     _setCanvasProperties(canvas);
 
+    _addPolygon(canvas);
     // Adding a simple rectangle to canvas
     _addRectangle(canvas);
-
+    _addCircle(canvas);
     // Object controls not working until common selection
     // Issue found when importing pdfs
-    _workaround(canvas);
```

Comment:
This fixes the issues with objects not being able to be deleted via the delete custom control. Should be kept as it is.